src/plugins/data/common/types.ts

```diff
@@ -18,3 +18,4 @@
  */
 
 export * from './timefilter/types';
+export * from './query/types';
```

Comment:
Due to the needs https://github.com/elastic/kibana/pull/44039#discussion_r322480652, please, could you also export expressions too?

Reply:
Expressions were moved to a separate plugin.
The types are available at `src/plugins/expressions`